srcpkgs/ardour/patches/6.0-fix-runtime.patch

```diff
@@ -1,9 +1,9 @@
 --- libs/pbd/pbd/pthread_utils.h.orig	2020-09-18 22:30:18.353441907 +0700
 +++ libs/pbd/pbd/pthread_utils.h	2020-09-18 22:37:07.059363532 +0700
-@@ -54,7 +54,7 @@
- # define PBD_RT_STACKSIZE_PROC 0x20000 // 128kB
- #endif
+@@ -49,7 +49,7 @@
+ #include "pbd/signals.h"
  
+ #define PBD_RT_STACKSIZE_PROC 0x80000 // 512kB
 -#define PBD_RT_STACKSIZE_HELP 0x08000 // 32kB
 +#define PBD_RT_STACKSIZE_HELP PBD_RT_STACKSIZE_PROC // 32kB
```

Comment:
Maybe you should fix the comment to be 512kB, I think?

Reply:
thats the thing... `PBD_RT_STACKSIZE_PROC` was 128kB before thus `PBD_RT_STACKSIZE_HELP` was too but now they've increased the former to 512kB and I'm not sure what the latter should be

Reply:
oh you mean the comment yeah right